test/parallel/test-fs-read-promises-optional-params.js

```diff
@@ -2,26 +2,56 @@
 
 const common = require('../common');
 const fixtures = require('../common/fixtures');
-const fs = require('fs');
-const read = require('util').promisify(fs.read);
-const assert = require('assert');
-const filepath = fixtures.path('x.txt');
-const fd = fs.openSync(filepath, 'r');
+const fs = require('node:fs');
+const fsPromises = require('node:fs/promises');
+const read = require('node:util').promisify(fs.read);
+const assert = require('node:assert');
+const { describe, test } = require('node:test');
```

Comment:
```suggestion
const { describe, it } = require('node:test');
```


"it" is the correct function for "describe", but I don't remember if it really matters under the hood

Reply:
I think one is an alias of the other, but changed it anyway